### PR TITLE
fix(DB/SAI): Eskhandar Combat Script

### PR DESCRIPTION
**Pull request by Voxstrasza**
## Changes Proposed:
- Script Eskhandar

## SOURCE:

https://www.wowhead.com/npc=14306/eskhandar
https://www.youtube.com/watch?v=BHiVWoPUhwI

## How to Test the Changes:

.additem 18203 & 18202
hit something until it spawns

### CODE REVIEW

**Comment by Voxstrasza** [comment]
dont know why its failing

> ERROR 1064 (42000) at line 29441: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'DELETE FROM `creature_template_addon` WHERE (`entry` = 14306)' at line 5

File: data/sql/updates/pending_db_world/rev_1639287017095533600.sql [hunk 1]
```diff
@@ -0,0 +1,18 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES 
+
+/* Eskhandar Ghost Effect
+*/
+
+DELETE FROM `creature_template_addon` WHERE (`entry` = 14306);
+INSERT INTO `creature_template_addon` (`entry`, `path_id`, `mount`, `bytes1`, `bytes2`, `emote`, `visibilityDistanceType`, `auras`) VALUES
+(14306, 0, 0, 0, 0, 0, 0, 22650);
```
Comment (Kitzunu) [inline data/sql/updates/pending_db_world/rev_1639287017095533600.sql:8]: auras is a string, rap it in quotes

File: data/sql/updates/pending_db_world/rev_1639287017095533600.sql [hunk 2]
```diff
@@ -0,0 +1,18 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES
```
Comment (Kitzunu) [inline data/sql/updates/pending_db_world/rev_1639287017095533600.sql:1]: this

**Comment by yuanf225** [comment]
> dont know why its failing
> 
> > ERROR 1064 (42000) at line 29441: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'DELETE FROM `creature_template_addon` WHERE (`entry` = 14306)' at line 5

Pay attention to your MySQL/MariaDB version, MariaDB 10.5 and above are no problem

**Comment by claudiodfc** [comment]
Test result 👍🏽 

image

### Note
- Stayed in combat for a long while until decided to reset, since no more than 2 spawned, which I guess is from each glove.
- After reset they came back. Not sure if just bad odds but It looks like it won't trigger until combat is reset?

**Comment by Voxstrasza** [comment]
according to comments from retail, the Eskhandar is supposed to last for 2 minutes but the spell to summon him is on a 1 minute cooldown - in theory you could summon two back to back but never two at one time.  can you clarify on your tests please

**Comment by claudiodfc** [comment]
Exatcly what you just said, I got two spawned at once. However, once they despawned I never got a single one back until I reset combat.

**Review by Nyeriah — APPROVED** [review]

**Comment by Voxstrasza** [comment]
> Exatcly what you just said, I got two spawned at once. However, once they despawned I never got a single one back until I reset combat.

that's an issue with something else then - this PR just scripts the combat routine of the NPC

**Pull request merged**